addons/mail/static/tests/activity/activity_menu_tests.js

```diff
@@ -0,0 +1,25 @@
+/** @odoo-module **/
+
+import { ActivityMenu } from "@mail/activity/activity_menu";
+import { click, getFixture, mount } from "@web/../tests/helpers/utils";
+import { makeMessagingEnv, MessagingServer } from "../helpers/helpers";
+
+let target;
+
+QUnit.module("mail", (hooks) => {
+    hooks.beforeEach(async () => {
+        target = getFixture();
+    });
+
+    QUnit.module("activity menu");
+
+    QUnit.test("activity menu: no activity", async (assert) => {
+        const server = new MessagingServer();
+        const env = makeMessagingEnv((route, params) => server.rpc(route, params));
+        await mount(ActivityMenu, target, { env });
+        await click(document.querySelector("i[aria-label='Activities']").closest('.dropdown-toggle'));
+        assert.containsOnce(target, ".o-mail-no-activity");
+        assert.strictEqual(target.querySelector(".o-mail-no-activity span").innerText, "Congratulations, you're done with your activities.");
```

Comment:
Would not assert explicit text (the classname `o-mail-no-activity` is good enough)